### DISCUSSION TITLE
Mysql - my.cnf file location

### DISCUSSION THREAD

**Discussion by monoloco64856** [caprover/caprover#1513] [Q&A] [answered]
Hi,

I need to add some stuff to my.cnf Mysql. Where is located Mysql files ?

Best regards

**Comment by maietta** [comment, marked as answer]
This is not related to Caprover.

**Reply by monoloco64856** [reply]
Mysql is installed on CapRover and I can't locate Mysql's app files on my droplet

**Reply by maietta** [reply]
Are you familiar with how containers work?

CapRover is a tool to help you deploy containerized applications. Your MySQL app lives in a container, meaning you'll want to probably create a host mounted file path mapped to the container path MySQL is expecting for the location of your config. You can do this in CapRover, but you'll need to locate your app's config inside the container, copy the contents and create the same file somewhere on your host, then map them in CapRover. You'll probably have to restart your MySQL container after making any changes to the config.

Again, this has nothing to do with CapRover itself.
